experimental/Schemes/ToricBlowups/attributes.jl

```diff
@@ -13,7 +13,7 @@ Normal toric variety
 julia> blow_down_morphism = blow_up(P3, [0, 1, 1])
 Toric blowdown morphism
 
-julia> underlying_morphism(blow_down_morphism)
```

Comment:
@HechtiDerLachs I mean this here for example. I just wanted to double check that you intend to not export those functions.

Reply:
Yes, `underlying_morphism` is internal. @HereAround and I already discussed that the user-facing counterpart should be `forget_toric_structure` in this particular case. `underlying_morphism`, on the other hand, is for forwarding various functionality through the type hiecharchy and applies to far more concrete types than just toric blow-down morphisms.